src/gsi-ncdiag/gsi_ncdiag.py

```diff
@@ -1362,7 +1383,8 @@ def toIODAobs(self, OutDir, ObsBias, QCVars, TestRefs, clobber=True):
         gsiqcname = value, 'GsiEffectiveQC'
         errname = value, 'GsiFinalObsError'
         gsiqc = np.zeros_like(outdata[varDict[value]['valKey']])
-        gsiqc[outdata[errname] > 1e8] = 1
```

Comment:
This is a 'special' variable we added (GsiEffectiveQC) with the intention that it would be binary, there should already be 'QC_Flag' used somewhere I thought...

Reply:
Thanks.  I noticed there was a variable "PreQC" but I couldn't understand how it was saved from "QC_flag" then, so I modified this one. Anyway, I removed this change in the updated PR.